tests/python/pants_test/backend/python/rules/test_pex.py

```diff
@@ -47,15 +53,17 @@ def setUp(self):
     super().setUp()
     init_subsystems([PythonSetup, PythonNativeCode, SubprocessEnvironment])
 
-  def create_pex_and_get_all_data(self, *, requirements=None, entry_point=None, interpreter_constraints=None,
+  def create_pex_and_get_all_data(self, *, requirements=PexRequirements(),
+      entry_point=None,
+      interpreter_constraints=PexInterpreterContraints(),
```

Comment:
If you take my recommendation about interpreter constraints being optional, I would still recommend this being `None`